src/simplebar.js

```diff
@@ -488,6 +489,22 @@ export default class SimpleBar {
         this.removeListeners();
         this.el.SimpleBar = null;
     }
+    
+    /**
+     * Destroy the object and remove traces of it in dom. 
+     * Restore element to original parent.
+     */
+    destroy() {
+        this.unMount();
+        this.originalParent.apendChild(this.el);
+        this.originalParent.querySelector('[class*="simplebar-"]').remove();
```

Comment:
This will select only 1 node and delete as well only 1. If we want to delete every node with class `simplebar-` you'll need to use something like `querySelectorAll` instead and loop through the results to remove them all.